_posts/2021-11-05-teaching-kit.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Developing teaching material"
+date:   2021-11-05 14:30:00 +0200
```

Comment:
The date of this post needs to be after the ones it links to.  Link to posts using the [post_url](https://jekyllrb.com/docs/liquid/tags/#linking-to-posts) tag.

Reply:
Correcting this. Confused about whether the url of the post should be the github url or the one of the published page. I suppose the latter?

Reply:
Follow the link in the comment for post_url. That describes how to link to other posts using the name of the file. When the site is rendered, the markup will be replace by the correct url